src/vehicleData.js

```diff
@@ -7,12 +7,12 @@
  * to be regenerated each time
  */
 import _ from "lodash";
-import parsedJsonData from "./rawData";
-const rawLogs = _.reverse(parsedJsonData.rawLogs);
-const jwt = parsedJsonData.jwt;
-const projectId = parsedJsonData.projectId;
-const apikey = parsedJsonData.APIKEY;
-const solutionType = parsedJsonData.solutionType;
+import { parsedData } from "./rawData";
+const rawLogs = _.reverse(parsedData.rawLogs);
+const jwt = parsedData.jwt;
+const projectId = parsedData.projectId;
+const apikey = parsedData.APIKEY;
```

Comment:
Is the capitalization here a naming pattern?

Reply:
Clearly there are consistency issues.  This repo is setup to enforce most formatting rules with eslint.  But I don't have something similar to enforce naming.    I think APIKEY is the odd-one out,  I'll file an issue to look at cleaning it up (https://github.com/googlemaps/fleet-debugger/issues/69)